### PR TITLE
Ref/specs

### PR DESCRIPTION
**Pull request by dsludwig**
Fixes Anaconda-Server/Anaconda-Server#124 #290 

Depends on Anaconda-Server/Anaconda-Server#1959

The `groups` command did not really work properly, and gave terrible error messages. Clean it up a bunch.

### CODE REVIEW

**Comment by rafaelhbarros** [comment]
group spec :+1:

File: binstar_client/utils/spec.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+        return self._group_name
+
+    @property
+    def member(self):
+        if self._member is None:
+            raise UserError('Member name not given (got %r expected <organization>/<group_name>/<member>)' % (self.spec_str,))
+        return self._member
+
+
+def group_spec(spec):
```
Comment (rafaelhbarros) [inline binstar_client/utils/spec.py]: +1

**Comment by rafaelhbarros** [comment]
What tests verify this?

**Comment by rafaelhbarros** [comment]
Build is not passing, I restarted the osx on 3.4

**Comment by dsludwig** [comment]
@rafaelhbarros tests added, thanks for pointing that out!

**Comment by rafaelhbarros** [comment]
build failing on osx again.

**Comment by dsludwig** [comment]
OS X is flaky - I ran it again, and it succeeded. Are we good to go?

**Pull request merged**